### PR TITLE
File Added : Assigment 7&8

### PR DESCRIPTION
**Pull request by Amit727619**

### CODE REVIEW

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+          
+        </style>
+    </head>
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_2 .html]: Remove extra line

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                <button onclick="show()" class="btn">GET</button>
+                <div class="box">
+                    <h3>output is:</h3>
+                    <center>
+                         <h4 id="pid"></h4>
+                    </center>
+                </div>
+            </div>
+        </body>
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_2 .html]: Remove extra line

File: amit/Javascript/Assigment7/Assigment7_4.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                }
+                document.getElementById("pid2").innerHTML += "<br>*****After Sorting******<br>";
+                for (j = 0; j < a; j++) {
+                    document.getElementById("pid2").innerHTML += "Marks " + (j + 1) + ":&nbsp;" + c[j] + "<br>";
+                }
+                document.getElementById("pid2").innerHTML += ` The sorted array : ${c} ` ;
+            }
+        </script>
+        <style>
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_4.html]: Remove extra line

File: amit/Javascript/Assigment7/Assigment7_4.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            <center>
+                <h4 id="pid"></h4>
+                <h4 id="pid1"></h4>
+                <h4 id="pid2"></h4>
+            </center>
+        </div>
+    </div>
+</body>
+
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_4.html]: Remove extra lines

File: amit/Javascript/Assigment7/Assigment7_5.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+                background-color: red;
+                outline-color: white;
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_5.html]: Remove extra line

File: amit/Javascript/Assigment7/Assigment7_5.html [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+                box-sizing: 30px;
+            }
+            
+        </style>
+    </head>
+   
+     
+      
+
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_5.html]: Remove all extra lines

File: amit/Javascript/Assigment7/Assigment7_5.html [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+            <center>
+                <h2>For Updating Marks students</h2>
+                <hr />
+            </center>
+
+            <label>How many students</label>
+            <input type="number" id="num1" placeholder="Enter the First number" />
+
+            <button onclick="show()" class="btn">GET</button>
+
```
Comment (pooja-nagare) [inline amit/Javascript/Assigment7/Assigment7_5.html]: Remove extra line

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+                background-color: red;
+                outline-color: white;
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+
```
Comment (Pawan1005) [inline amit/Javascript/Assigment7/Assigment7_2 .html]: remove extra line

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+          
+        </style>
+    </head>
+         <body>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment7/Assigment7_2 .html]: create proper indentation

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+                <button onclick="show()" class="btn">GET</button>
+                <div class="box">
+                    <h3>output is:</h3>
+                    <center>
+                         <h4 id="pid"></h4>
+                    </center>
+                </div>
+            </div>
+        </body>
+    </body>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment7/Assigment7_2 .html:62]: remove extra body tag

File: amit/Javascript/Assigment7/Assigment7_2 .html [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+</html>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment7/Assigment7_2 .html]: add new line at the end of file

File: amit/Javascript/Assigment8/Assigment8_1.html [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+                background-color: red;
+                outline-color: white;
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+        </style>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment8/Assigment8_1.html:29]: put style tag in head part

File: amit/Javascript/Assigment8/Assigment8_2.html [hunk 13]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```
Comment (Pawan1005) [inline amit/Javascript/Assigment8/Assigment8_2.html]: remove extra line

File: amit/Javascript/Assigment8/Assigment8_2.html [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+        </style>
+        <center><h1>Quick Sort</h1></center>
+
```
Comment (Pawan1005) [inline amit/Javascript/Assigment8/Assigment8_2.html]: remove extra line

File: amit/Javascript/Assigment8/Assigment8_3.html [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+                background-color: red;
+                outline-color: white;
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+        </style>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment8/Assigment8_3.html]: put style tag in head part

File: amit/Javascript/Assigment8/Assigment8_4.html [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+                background-color: red;
+                outline-color: white;
+                outline-width: 15px;
+                color: white;
+            }
+            .label {
+                border-radius: 15px;
+                box-sizing: 30px;
+            }
+        </style>
```
Comment (Pawan1005) [inline amit/Javascript/Assigment8/Assigment8_4.html:32]: put style tag in head part

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**